File: Pocs/Automated Test/AgileContent/Node.js/SeleniumMocha/Personal/Pages/home-page.js
const {By} = require('selenium-webdriver');
const BasePage = require('./base-page');

class HomePage extends BasePage {

  constructor(driver) {
    super(driver);
    this.ingresarLocator = By.xpath('//*[@id="__next"]/header/div/div[4]/button');
    this.perfilLocator = By.xpath('//*[@id="__next"]/header/div/div[4]/div/button');
    this.miContenidosLocator = By.xpath('//*[@id="dropMenu"]/div/div/div/div/ul/nav[1]/div');
  }

  async ingresar() {
    await this.getElement(this.ingresarLocator).click();
  }

  async goToMiBiblioteca() {
    await this.getElement(this.perfilLocator).click();   
    await this.getElement(this.miContenidosLocator).click();
  }
}

module.exports = HomePage;

File: Pocs/Automated Test/AgileContent/NetCoreStandard/NetCoreSeleniumTest/Pages/HomePage.cs
using OpenQA.Selenium;

namespace NetCoreSeleniumTest.Pages
{
    public class HomePage: BasePage
    {
        private By ingresarLocator;
        private By perfilLocator;
        private By miContenidosLocator;

        public HomePage(IWebDriver driver) : base(driver)
        {
            ingresarLocator = By.XPath("//*[@id='__next']/header/div/div[4]/button");
            perfilLocator = By.XPath("//*[@id='__next']/header/div/div[4]/div/button");
            miContenidosLocator = By.XPath("//*[@id='dropMenu']/div/div/div/div/ul/nav[1]/div");
        }

        public void Ingresar()
        {
            GetElement(ingresarLocator).Click();
        }

        public void GoToBiblioteca()
        {
            GetElement(perfilLocator).Click();

            GetElement(miContenidosLocator).Click();
        }
    }
}
